### ISSUE TITLE
[Raid Help] Marcador de Posiciones 

### ISSUE THREAD

**Issue by skyrex321** [RoDNet/Bugtracker-Infinium-4.0.6a#985]
Bueno ya se que a mucha gente no le importara esta bug ya que no lo suelen utilizar pero para mi y la mayoría de los que explicamos los bosses de las raids es un útil indispensable para que la gente lo entienda ya que como siempre se ha dicho una imagen vale mas que 1000 palabras
![Bug en Banderitas Raids](https://f.cloud.github.com/assets/2746480/344309/8095e832-9dfc-11e2-80cb-bbf6a5c15460.JPG)
 Como se ve en la imagen te deja seleccionar la banderita que quieres poner pero cuando lo pones sobre el terreno 
![Poniendo Marca](https://f.cloud.github.com/assets/2746480/344316/b195acce-9dfc-11e2-82cb-a983f41f33c2.JPG)
No se pone la Marca ni nada por el estilo a la hora de explicar los bosses en la que la gente debera moverse lo mejor es poder tener esta herramienta de la consola de raid.

Muchas Gracias espero que lo puedan arreglar

**Comment by crowel** [comment]
Cierto, estaria muy bien marcar las posiciones, para que despues en las raids no hubiera problemas de posicion.
